### ISSUE TITLE
Import PuTTy/KiTTy sessions 

### ISSUE THREAD

**Issue by Thunderhawk2001** [1Remote/1Remote#467]
**Is your feature request related to a problem? Please describe.**
It's pretty infuriating to have to manually add every Session from PuTTy/KiTTy into 1Remote.

**Describe the solution you'd like**
A way to import my current PuTTy/KiTTy sessions into 1Remote with "just one click" or via *.reg import of the Putty Sessions.

**Describe alternatives you've considered**
Only alternative would be to not use 1Remote. So we would continue to use PuTTy/KiTTy or another app, which features this import functionality.

**Comment by majkinetor** [comment]
If this is done, it should be done in idempontent way so that people can re-do import without duplicating previously imported connections, since you can create new Kitty connections after importing existing ones in 1RM.

**Comment by VShawn** [comment]
there may no way to export putty's sessions. putty does not provide export feature, and mRemoteNG only read the session name from putty without importing putty's sessions into their own database.

**Comment by Thunderhawk2001** [comment]
Oh, okay.

I've used a work around now:
I downloaded mRemoteNG, started it, created a dummy SSH entry, exported that as *.CSV. Now I opened this file with excel and pasted my list of ssh servers as name and hostname, used a new ID starting with 1, and kept every other option the same as the dummy entry. 
Now I could import this list via the "import mRemoteNG csv" function.

Screenshot:
![grafik](https://github.com/1Remote/1Remote/assets/14139482/716dd980-2527-4f64-b205-19601ce53d1e)
